modules/ai/assets/js/editor/hooks/use-prompt.js

```diff
@@ -1,12 +1,13 @@
 import { useState } from 'react';
 import { setStatusFeedback } from '../api';
 
-const normalizeResponse = ( { text, response_id: responseId, usage, images } ) => {
+const normalizeResponse = ( { text, response_id: responseId, usage, images, base_template_id: baseTemplateId } ) => {
```

Comment:
`base_template_id` is `undefined` for requests that aren't from the AI Builder

Well... It's not your fault, it's because we've used this function to normalize everything, but maybe we need to separate it somehow? WDYT? (maybe also a different PR)

Reply:
I think that this area needs a small refactor...
How it's now?

Reply:
looks better
probably still requires a refactor, but that's for another PR